util/drakeUtil.cpp

```diff
@@ -182,3 +183,71 @@ void sizecheck(const mxArray* mat, int M, int N) {
   }
   return;
 }
+
+void callMexStandalone(const char* mexFile, int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[]) {
```

Comment:
do we still want this hanging around?  (does mexwrapper supercede it?)  or at very least, can we move it over to mexwrapper, and add some documentation to describe the two options to people that are poking around in the code?